### DISCUSSION TITLE
What are the differences for each model letter 'type'?

### DISCUSSION THREAD

**Discussion by afbagwell** [Bing-su/adetailer#282] [Q&A] [answered]
The instructions were very unclear on this point. There are several face (and hand) yolo models that are only different in a single letter. What do these mean?

8n
8s
8m

Are these related to size of the face relative to the picture? Level of detail? Why should I pick one over the other?

**Comment by Symbiomatrix** [comment, marked as answer]
As far as I know, it's just the size of the pretrained model: m > s > n. Other than possibly vram concern, m has the highest accuracy (it's listed in the readme as mAP = mean average precision). Since we're not talking huge models, overfitting and questionable improvement, I don't see much reason to run the smaller ones other than availability for each object.
